lighthouse/lighthouse.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-boot_nodes=$(curl -s https://config.dencun-devnet-9.ethpandaops.io/api/v1/nodes/inventory | jq '[ .ethereum_pairs[] | .consensus.enr ] | join(",")' | tr -d '"')
+boot_nodes=$(echo /config/testnet/boot_enr.yaml | jq '. | join(",")' | tr -d '"')
```

Comment:
I think we can probably get rid of passing the `boot_nodes` here, lighthouse already reads the boot_enr.yaml file in the testnet directory. I'll try without this and see if it works.

Reply:
Main reason for this is now we've got a much bigger list of ENRs and having them all passed as parameters may not work well.

Reply:
I removed it and it seems to be working fine , syncing..